extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -428,4 +434,18 @@ private function getTrendingVideosData() {
 
 		return $data;
 	}
+
+	private function getWikiStats() {
+		global $wgCityId;
+
+		$wikiStats = [];
+		$service = new WikiDetailsService();
+		$wikiDetails = $service->getWikiDetails( $wgCityId );
+
+		if ( !empty( $wikiDetails['stats'] ) ) {
```

Comment:
We won't reference key from non existent object after you change https://github.com/Wikia/mercury/pull/881/files#r31415277